templates/shared/secrets/add_list.tmpl

```diff
@@ -62,7 +65,6 @@
 				<input autofocus required
 					id="secret-name"
 					name="name"
-					value="{{.name}}"
```

Comment:
Why remove this?

Reply:
Is this property actually needed? I don't see it used anywhere, and the template isn't setting its value.